config.yaml

```diff
@@ -22,6 +22,7 @@ repos:
     url: https://github.com/coreos/afterburn
     vars:
       git_repo: afterburn
+      rhel_package_repo: rust-afterburn
```

Comment:
Sorry, I was slow to notice this: the new field shouldn't be needed at all, because the dist-git repo name always matches the specfile name.  So the next commit can just use the existing `rhel8_package` variable.

Reply:
No worries lol I did not notice that either! thank you for catching that and allowing use not to pollute our vars.

Reply:
Hm, just noticed there was no mention of console-login-helper-messages has no value for rhel8_package, but in my searching for the gitlab rpm repos I did see a package there. Should I just add a value for console-login-helper-messages's rhel8_package?

Reply:
That repo doesn't have a release checklist, so the value wouldn't be used.  I'd be inclined to omit, to keep things simple.